AIR_Wheelly_BLL/Services/AuthService.cs

```diff
@@ -118,5 +118,46 @@ public string GenerateJwtToken(int Id)
             return user;
 
         }
+
+        public async Task<User?> UpdateProfileAsync(UpdateProfileDTO dto, string jwtToken)
+        {
+            var handler = new JwtSecurityTokenHandler();
+            var token = handler.ReadToken(jwtToken) as JwtSecurityToken;
```

Comment:
ja bi ovo stavio u helper klasu za jwt, jer cemo to cesto koristit pa da se ne ponavljamo